src/App.vue

```diff
@@ -1,48 +1,64 @@
+        </div>
 
-      <div class="grid col-start-6 col-end-8 justify-items-end sm:col-start-7">
-        <!-- named routes - https://router.vuejs.org/guide/essentials/named-routes.html -->
-        <!-- logout -->
-        <div v-if="showLogout" class="text-lg sm:text-xl">
-          <router-link v-if="!isAuthenticated" :to="{ name: 'Login' }">
-            <button class="bg-white-500 hover:text-red-500 text-black font-bold border-0 object-contain">
-              {{ $t("nav.login") }}
-            </button>
-          </router-link>
-          <a href="#" v-if="isAuthenticated" @click="logoutButtonClicked">
-            <button class="bg-white-500 hover:text-red-500 text-black font-bold border-0 object-contain px-1 py-2">
-              {{ $t("nav.logout") }}
-            </button>
-          </a>
+        <div class="grid col-start-6 col-end-8 justify-items-end sm:col-start-7">
+          <!-- named routes - https://router.vuejs.org/guide/essentials/named-routes.html -->
+          <!-- logout -->
+          <div v-if="showLogout" class="text-lg sm:text-xl">
+            <router-link v-if="!isAuthenticated" :to="{ name: 'Login' }">
+              <button class="bg-white-500 hover:text-red-500 text-black font-bold border-0 object-contain">
+                {{ $t("nav.login") }}
+              </button>
+            </router-link>
+            <a href="#" v-if="isAuthenticated" @click="logoutButtonClicked">
+              <button class="bg-white-500 hover:text-red-500 text-black font-bold border-0 object-contain px-1 py-2">
+                {{ $t("nav.logout") }}
+              </button>
+            </a>
+          </div>
+          <!-- locale switcher -->
+          <div class="self-center">
+            <LocaleSwitcher id="locale" class="flex justify-center"></LocaleSwitcher>
+          </div>
         </div>
-        <!-- locale switcher -->
-        <div class="self-center">
-          <LocaleSwitcher id="locale" class="flex justify-center"></LocaleSwitcher>
+      </div>
+      <router-view />
+    </div>
+    <!-- first-time language picker -->
+    <div class="fixed w-full mt-40 flex justify-center" v-if="showLanguagePicker">
+      <div class="bg-white w-11/12 sm:w-9/12 lg:w-7/12 p-4 sm:p-10 rounded-lg border border-black">
+        <p class="text-center text-2xl sm:text-4xl py-4 sm:py-8">Select your language</p>
+        <div class="grid grid-cols-2 space-x-2">
+          <div class="bg-green-400 hover:bg-green-600 p-4 sm:p-8 rounded-lg">
+            <p class="text-xl sm:text-3xl cursor-pointer text-white" @click="setLocale('en')">English</p>
+          </div>
+          <div class="bg-red-400 hover:bg-red-600 p-4 sm:p-8 rounded-lg">
+            <p class="text-xl sm:text-3xl cursor-pointer text-white" @click="setLocale('hi')">हिंदी</p>
+          </div>
         </div>
       </div>
```

Comment:
Also, the dialog box looks weird in landscape mode
And I can't even scroll down. the dialog just stays there half cut

<img width="748" alt="Screenshot 2021-06-02 at 10 33 19 AM" src="https://user-images.githubusercontent.com/20169829/120426574-f9948f80-c38d-11eb-9471-4caf9bf03457.png">


https://user-images.githubusercontent.com/20169829/120426634-15983100-c38e-11eb-875b-1cbace252669.mov